.github/workflows/generate-summary.yaml

```diff
@@ -406,6 +406,20 @@ jobs:
             -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
             https://api.github.com/rate_limit
 
+      - name: Create Issue Gist
+        run: |
+          python3 -m pip install pygithub requests pyopenssl --upgrade
+          FILE_SIZE=$(stat -c%s "issue.md")
+          echo "file_size : $FILE_SIZE"
+          ONE_MB_IN_BYTES=1048576
+          head -c $ONE_MB_IN_BYTES "issue.md" > gist_trimmed_issue.md
+          if [ "$FILE_SIZE" -gt "$ONE_MB_IN_BYTES" ]; then
+              echo "The file was greater than 1 MB and has been trimmed to fit the gist limit. Please check logs for the untrimmed issue."
+          fi
+          python3 ./scripts/create_gist.py --token ${{ secrets.GITHUB_TOKEN }} --input gist_trimmed_issue.md --output issue_gist_url.txt
+          cat issue_gist_url.txt
+        continue-on-error: true
+
       - name: Trim issue length # reduce the number of lines in final issue so github always creates issue
         run: |
           head -c 65000 issue.md > trimmed_issue.md
```

Comment:
Can we emit the URL as a part of the "Issue text has been trimmed" message?

Reply:
ditto